pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -62,10 +61,16 @@ const (
 	npSyncReadyBookMarkCnt = 3
 )
 
+var cloudSyncInterval *int64
```

Comment:
dont declare global variables in each package.. Make controller config as global, and use Getter method in individual packages

Reply:
See how ControllerSyncStatus is global and accessed from individual controller

Reply:
Solved